apptools/apptools-android-tests/apptools/CI/crosswalk_pkg_basic.py

```diff
@@ -45,16 +47,32 @@
 apkLength = 0
 
 class TestCrosswalkApptoolsFunctions(unittest.TestCase):
+    def invokeCrosswalkPkg(self, params=None):
+        params = params or []
+        cmdline = []
+        if comm.HOST_PREFIX:
+            cmdline.append(comm.HOST_PREFIX)
+        cmdline.extend([os.path.join(comm.PackTools, 'crosswalk-pkg')])
+        cmdline.extend(params)
+        process = subprocess.Popen(cmdline, stdout=subprocess.PIPE,
+                                   stderr=subprocess.STDOUT)
+        output, _ = process.communicate()
+        self.assertEquals(process.returncode, 0)
+        return output
 
     def test_path_absolute(self):
         comm.setUp()
         os.chdir(comm.XwalkPath)
         comm.clear("org.xwalk.test")
         os.mkdir("org.xwalk.test")
         os.chdir('org.xwalk.test')
-        cmd = comm.HOST_PREFIX + comm.PackTools + \
-            "crosswalk-pkg --platforms=android --android=" + comm.ANDROID_MODE + ' --targets="' + comm.BIT + '" ' + comm.ConstPath + "/../testapp/create_package_basic/"
-        (return_code, output) = comm.getstatusoutput(cmd)
+        output = self.invokeCrosswalkPkg([
+            '--platforms=android',
+            '--android=' + comm.ANDROID_MODE,
+            '--crosswalk=' + comm.crosswalkzip,
+            '--targets=' + comm.BIT,
+            comm.ConstPath + '/../testapp/create_package_basic/',
+        ])
```

Comment:
def test_path_absolute(self) checkpoint: default is stable without --crosswalk, and app path support absolute path, if package command with --crosswalk, need remove 
version = comm.check_crosswalk_version(self, "stable")
I will add testcase to test default is stable without --crosswalk